rust/src/sysroot_upgrade.rs

```diff
@@ -56,10 +56,32 @@ fn layer_counts<'a>(layers: impl Iterator<Item = &'a ManifestLayerState>) -> (u3
     )
 }
 
+// Output when we begin/end fetching a layer.  Ideally, we'd handle
+// byte-level progress here too, but that requires some more sophisticated
+// binding with the rpmostree-output.h via cxx.rs.
 async fn layer_progress_print(mut r: Receiver<ImportProgress>) {
+    // This is just used to hold a reference to the task.
+    #[allow(unused_variables, unused_assignments)]
+    let mut task = None;
     while let Some(v) = r.recv().await {
         let msg = ostree_ext::cli::layer_progress_format(&v);
-        output_message(&msg);
+        tracing::debug!("layer progress: {msg}");
+        match v {
+            ImportProgress::OstreeChunkStarted(_) => {
```

Comment:
One other tech debt here is that this would all blow up if we started fetching layers concurrently.  Ultimately we're going to switch to using podman via https://github.com/containers/bootc/pull/215 so we can ignore this.